### PR TITLE
Theme init

### PR DESCRIPTION
**Pull request by bludot**

### CODE REVIEW

**Comment by bludot** [comment]
Will add rollup later for es modules support

File: examples/main/package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
   "dependencies": {
     "react": "^16.5.2",
     "react-dom": "^16.5.2",
-    "ui-framework": "git+https://github.com/selfapy/ui-framework.git#init"
+    "styled-components": "^3.4.10",
+    "ui-framework": "git+https://github.com/selfapy/ui-framework.git"
```
Comment (prutya) [inline examples/main/package.json]: You can just `"ui-framework": "github:selfapy/ui-framework"`

File: webpack.config.js [hunk 2]
```diff
@@ -29,8 +29,9 @@ module.exports = {
   resolve: {
     alias: {
       react: path.resolve(__dirname, './node_modules/react'),
+      'styled-components': path.resolve(__dirname, './node_modules/styled-components'),
```
Comment (prutya) [inline webpack.config.js]: Can we utilize [babel-plugin-module-resolver](https://www.npmjs.com/package/babel-plugin-module-resolver) here?

File: examples/main/package.json (hunk 1 above)
Comment (bludot) [reply examples/main/package.json]: Not if im taking from this branch. I will change when we merge in to master. Just master does not have the same setup as init.

File: webpack.config.js (hunk 2 above)
Comment (bludot) [reply webpack.config.js]: well, its not the webpack way and we dont bundle with styleguide and react.

**Comment by bludot** [comment]
Not needed

**Pull request closed without merging**